frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts

```diff
@@ -88,20 +84,17 @@ export const findVMIPod = (
 export const getVMImporterPods = (
```

Comment:
this function should return only pods that belong to or the vm is dependent on.

So we should return only pods which import VM's DV/PVC volumes

Reply:
That makes a lot of sense. =) Fixed!